Vagrantfile

```diff
@@ -52,29 +54,41 @@ Vagrant.configure(2) do |config|
         end
     end
 
-    # system setup for patroni ndoes
+    # system setup for patroni nodes
     (patroni_nodes).each do |node|
         config.vm.define node do |conf|
-            args = [ '-n', node, '-v', pgver ];
+            args_pa = [ '-n', node, '-v', pgver ];
+            args_vm = [ '-n', node, '-v', vmver, '-i', vip_ip.to_s ];
+
             (patroni_nodes).each do |p|
-                args.push("-p", "#{p}=#{nodes_ips[p]}")
+                args_pa.push("-p", "#{p}=#{nodes_ips[p]}")
             end
             (etcd_nodes).each do |e|
-                args.push("-e", "#{e}=#{nodes_ips[e]}")
+                args_pa.push("-e", "#{e}=#{nodes_ips[e]}")
+                args_vm.push("-e", "#{e}=#{nodes_ips[e]}")
             end
-            
+
             conf.vm.network 'private_network', ip: nodes_ips[node]
             conf.vm.provision 'patroni-setup', type: 'shell', path: 'provision/patroni.bash',
-                args: args,
+                args: args_pa,
```

Comment:
could be:

~~~
            args: args + [ '-v', pgver ];
~~~

Reply:
done